app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -152,4 +163,33 @@ private void storeStudentImage(Mat img){
 
     }
 
+    private void createOverlay() {
+        // Load overlay mask (to be completed...)
+        imgOverlay = Imgcodecs.imread(MultimediaHelper.getImageDirectory() + "/deer.jpg", Imgcodecs.IMREAD_UNCHANGED);
```

Comment:
We faced some problems using a transparent .png it didn't separated the back- and foreground as expected, so we tried with .jpegs where the background is white instead of transparent.